src/connector.js

```diff
@@ -1017,6 +1036,18 @@ function updateResultListState( newState ) {
 			else {
 				breadcrumb = '<p class="location"><cite><a href="' + clickUri + '">' + printableUri + '</a></cite></p>';
 			}
+			
+			let disp_declared_type = "";
```

Comment:
Please move this variable declaration just under the breadcrumb variable (around line 1026), so that all variable are grouped together.

Reply:
Moved